Discovery/discoveryplugin.py

```diff
@@ -307,9 +309,17 @@ def read_config(self):
         self.echosearchcolumn = settings.value("echo_search_column", True, type=bool)
         self.postgisdisplaycolumn = settings.value("display_columns", "", type=str)
         self.postgisgeomcolumn = settings.value("geom_column", "", type=str)
+        try:
+            self.display_time = float(settings.value("display_time", "", type=str))*1000
+        except:
+            self.display_time = 5000
```

Comment:
In general in Python it is discouraged to use "except:" without specification what kind of error is being caught as it can lead to catching unwanted exceptions. Better to use "except ValueError:" (or what is the correct exception)